src/compose/ComposeBox.android.js

```diff
@@ -15,13 +15,7 @@ import type {
   Dimensions,
   GlobalState,
 } from '../types';
-import {
-  addToOutbox,
-  cancelEditMessage,
-  draftUpdate,
-  fetchTopicsForActiveStream,
-  sendTypingEvent,
-} from '../actions';
+import { addToOutbox, draftUpdate, fetchTopicsForActiveStream, sendTypingEvent } from '../actions';
```

Comment:
I guess you will want to rebase the changes to this file as iOS and Android got merged.

Reply:
Yup! rebased.